bin/run_ingestion.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+#email_recipient=***********
+#eva_dir=**************
+# Grab the variables from a config bash script
+source ~/.covid19dp_processing 2> /dev/null
+
+if [ -z "${email_recipient}" ] || [ -z "${eva_dir}" ];
+then
+  echo "run_ingestion.sh does not have access to email_recipient and eva_dir variables. Please set them above or populate ~/.covid19dp_processing"
+  exit 1
+fi
+
+tmp_dir=${eva_dir}/scratch
+software_dir=${eva_dir}/software/covid19dp-submission/production_deployments/
+project_dir=${eva_dir}/data/PRJEB45554
+lock_file=${project_dir}/.lock_ingest_covid19dp_submission
+
+#Check if the previous process is still running
+if [[ -e ${lock_file} ]];
+then
+  echo "processing in $(cat ${lock_file}) is still going. Exit"
+  exit 0
+fi
+
+# Check if there is an unfinished process
+valid_dir=${project_dir}/30_eva_valid
+
+for dir in ${valid_dir}/????_??_??_??_??_??;
+do
+  if [[ ! -e ${dir}/.process_complete ]];
+  then
+    current_date=$(basename ${dir})
+    break
+  fi
+done
+
+# If the current_date is set and if it isn't then create a new one
+if [ -z "$current_date" ];
+then
+  current_date=$(date --rfc-3339=second | cut -d '+' -f 1 | sed 's/[- :]/_/g' )
+fi
+
+processing_dir=${valid_dir}/${current_date}
+
+echo ${current_date} > ${lock_file}
+# Ensure that the lock file is delete on exit of the script
+trap 'rm ${lock_file}' EXIT
+
+mkdir -p ${processing_dir}
+export TMPDIR=${tmp_dir}
+
+${software_dir}/production/bin/ingest_covid19dp_submission.py \
+  --project-dir ${project_dir} --app-config-file ${software_dir}/app_config.yml \
+  --nextflow-config-file ${software_dir}/workflow.config \
+  --processed-analyses-file ${project_dir}/processed_analysis.txt --num-analyses 10000 \
+  --resume-snapshot ${current_date} \
+  >> ${processing_dir}/ingest_covid19dp.log \
+  2>> ${processing_dir}/ingest_covid19dp.err
+
+# Assess success
+process_exit_status=$?
+set -o pipefail && cat ${processing_dir}/ingest_covid19dp.log | grep 'submission_workflow.nf' | grep 'completed successfully'
```

Comment:
```suggestion
set -o pipefail && cat ${processing_dir}/ingest_covid19dp.log | grep 'submission_workflow.nf' | grep -i 'completed successfully'
```

Reply:
Don't think I need this grep to ignore case since nextflow always write it lowercase.